index.html

```diff
@@ -0,0 +1,176 @@
+             <p class="desc2">@Munyonyo Commonwealth Resort, Along Lake Victoria Shores</p>
+             <!-- <div class="header-icons"> <h2 class="header-icon"><img src="images/thumbs-up.png" alt="Like" class="thumbs-up"> 5000</h2>
+              <h2 class="header-icon"><img src="images/thumbs-up.png" alt="Like" class="thumbs-up"> 5000</h2>
+              <h2 class="header-icon"><img src="images/thumbs-up.png" alt="Like" class="thumbs-up"> 5000</h2>
+             </div> -->
+        </div>
+      
+
+                 <div class="program-wrapper">
+                  <div class="title-wrap">
+                  <h3 class="pro-header">Main Program</h3>
+                  <hr>
+                 </div>
+        
+                      <div class="main-program">
+          
+                        <div class="program-card">
+                          <img src="images/speech.png" alt="" class="pro-icon">
+                          <h4 class="program">Lecture</h4>
+                          <p class="pro-desc">Listern to the Speakers from various countries
+                          about the messages of sharing and openning></p>
+                        </div>
+                        <div class="program-card">
+                          <img src="images/inov2.png" alt="Innovators Exhibition" class="pro-icon">
+                          <h4 class="program">Inovators</h4>
+                          <p class="pro-desc">Listern to the Speakers from various countries
+                          about the messages of sharing and openning></p>
+                        </div>
+                        <div class="program-card">
+                          <img src="images/invest.png" alt="Investments Forum" class="pro-icon">
+                          <h4 class="program">Investers</h4>
+                          <p class="pro-desc">Listern to the Speakers from various countries
+                          about the messages of sharing and openning></p>
+                        </div>
+                        <div class="program-card">
+                          <img src="images/net.jpg" alt="Networking" class="pro-icon">
+                          <h4 class="program">Networks</h4>
+                          <p class="pro-desc">Listern to the Speakers from various countries
+                          about the messages of sharing and openning></p>
+                        </div>
+                       </div>
+
+                              <div class="advert">
+                                <p>Join WFH Dev's Annual global  2022</p>
+                              </div>
+                              <div class="advert1">
+                                <p>SEE THE WHOLE PROGRAM</p>
+                              </div>
+                 </div>
+        <div class="speaker-head">
+        <h3 class="speaker-header">Featured Speakers</h3>
+        <hr>
+      </div>
+        <div class="speakers"></div>
```

Comment:
- [ ] [Optional] Please decrease the height of the speaker image in the speakers section to match the given design.

![image](https://user-images.githubusercontent.com/68796232/183267776-32e35fff-ad47-43cb-8cb7-797357fe711c.png)

### ⚠️I marked this as optional because the last reviewer forgot to mention it so please consider doing it⚠️